app/src/main/java/com/nervousfish/nervousfish/activities/LoginActivity.java

```diff
@@ -19,29 +19,29 @@
 /**
  * Simple login activity class.
  */
-public final class LoginActivity extends Activity {
+public final class LoginActivity extends AppCompatActivity {
```

Comment:
But the other day you said that we shouldn't use `AppCompatActivity ` all the time xd